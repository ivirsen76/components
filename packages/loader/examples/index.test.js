import test from '../../../scripts/testExamples.js'

test(__dirname)
