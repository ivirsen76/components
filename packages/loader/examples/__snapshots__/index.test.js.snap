// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader default.js 1`] = `
<Loader
  loading={false}
  removeChildren={false}
  text="Loading"
>
  <button
    className="ui button"
    onClick={[Function]}
  >
    Click me
  </button>
</Loader>
`;

exports[`loader different_text.js 1`] = `
<Loader
  loading={false}
  removeChildren={false}
  text="Wait for loading data"
>
  <button
    className="ui button"
    onClick={[Function]}
  >
    Click me
  </button>
</Loader>
`;

exports[`loader remove_children.js 1`] = `
<Loader
  loading={false}
  removeChildren={true}
  text="Loading"
>
  <button
    className="ui button"
    onClick={[Function]}
  >
    Click me
  </button>
</Loader>
`;

exports[`loader without_text.js 1`] = `
<Loader
  loading={false}
  removeChildren={false}
  text={null}
>
  <button
    className="ui button"
    onClick={[Function]}
  >
    Click me
  </button>
</Loader>
`;
